challenge.md

```diff
@@ -31,19 +31,19 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
+Absolute. It is exactly where you will find it.<!-- Answer Starts Here -->
 
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->Relative.  When someone else views that font, they won't be able to find it if you use an absolute path on your computer.
 
 <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
-<!-- Answer Starts Here -->
+<!-- Answer Starts Here -->Relative, so everyone can use it.
```

Comment:
You're correct that we would use a relative path here, but again your explanation is not clear. The reason we use a relative path here is so that, if we were to bundle up our project (which contains the image) and send it to someone else, the link to the image would still be correct, and not break.